src/Controller/GameManager.js

```diff
@@ -0,0 +1,58 @@
+const OutputView = require('../View/OutputView');
+const InputView = require('../View/InputView');
+const Validator = require('../Model/Validator');
+const Quit = require('../libs/Quit');
+const { MESSAGE, OPTION } = require('../libs/const');
+
+class GameManager {
+  #computerAnswer;
+
+  constructor(compuerAnswer) {
+    this.#computerAnswer = compuerAnswer;
```

Comment:
```suggestion
    this.#computerAnswer = new ComputerAnswer();
```
constructor에서 copuerAnswer을 받아 오지 않고 이렇게 수정해도 괜찮을 것 같아요~
copuerAnswer 오타 있는 것 같네요!

Reply:
오타가 있었네요!ㅋㅋㅋ 얼른 수정해야겠어요~
저도 처음엔 `GameManager` 클래스의 생성자 함수에서 은총님 말씀처럼 코드를 작성했는데, 어떤 것이 더 알맞는 방법인지 몰라서 이것저것 고민을 했었어요!